app/services/ui_notifications/clean_expired.rb

```diff
@@ -0,0 +1,41 @@
+module UINotifications
+  class CleanExpired
+    def initialize(options = {})
+      if options[:blueprint].present?
+        blueprint = NotificationBlueprint.find_by(name: options[:blueprint])
```

Comment:
if you just need to check if the blueprint exists, you can use `NotificationBlueprint.where(name: options[:blueprint]).exists?` without loading the actual blueprint. Although i'm not sure you have to check if it exists, worst case no notifications will be deleted.